database/bfgd/postgres/postgres.go

```diff
@@ -965,6 +965,62 @@ func (p *pgdb) AccessPublicKeyDelete(ctx context.Context, publicKey *bfgd.Access
 	return nil
 }
 
+// BtcBlocksHeightsWithNoChildren returns the heights of blocks that have no
+// child blocks in our database, these represent possible forks that we have
+// not handled yet
```

Comment:
I think we could improve the clarity of this comment a bit, what do you think of the following?
```suggestion
// BtcBlocksHeightsWithNoChildren returns the heights of blocks stored in the
// database that do not have any children, these represent possible forks that
// have not been handled yet.
```